cpp/test/Ice/background/AllTests.cpp

```diff
@@ -115,88 +115,6 @@ allTests(TestHelper* helper)
     }
     cout << "ok" << endl;
 
-    cout << "testing locator... " << flush;
```

Comment:
I don't see the relevance of testing the locator and router with this test. See also #1893

Reply:
It would be good to get Benoit's input.

Reply:
Commented on #1893